README.md

```diff
@@ -13,19 +13,19 @@ dependencies: ["https://github.com/DataDog/serverless-plugin-datadog/blob/master
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue)](https://github.com/DataDog/serverless-plugin-datadog/blob/master/LICENSE)
 
 Datadog recommends the Serverless Framework Plugin for developers using the Serverless Framework to deploy their serverless applications.
-The plugin automatically enables instrumentation for your Python and Node.js applications to collect metrics, traces, and logs by:
+The plugin automatically enables instrumentation for your Python, Node.js, Ruby, Java, Go, and .NET applications to collect metrics, traces, and logs by:
```

Comment:
The custom runtime (which we support now) could use any language, not just Go. So maybe it would be better to just avoid listing specific languages here.